src/site/topic25.rst

```diff
@@ -18,9 +18,55 @@ Topic #25 --- Sorting
 
 * `Here is a simple visualization of some popular sorting algorithms <https://www.toptal.com/developers/sorting-algorithms>`_
 
+
 Sorting Problem
 ===============
 
+* Given a collection of *things*
+    * Numbers
+    * Strings
+    * People
+    * Car Parts
+
+* The goal is to arrange the collection of *things* such that they are *in order*
+
+
+Order
+-----
+
+* What *in order* means however will depend on what is being sorted
+* For example
+    * Numbers come with an intrinsic ordering
```

Comment:
Java uses [natural ordering](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/Comparable.html) to describe this.

> This ordering is referred to as the class's _natural ordering_, and the class's `compareTo` method is referred to as its _natural comparison method_.